### ISSUE TITLE
Stand Alone png

### ISSUE THREAD

**Issue by pssguy** [palaeoverse/rphylopic#1]
Great work Scott

Is it possible to create a png of the silhouette?

**Comment by cboettig** [comment]
Agree, a stand alone png (as file, etc) would be a great feature.  

Current use cases illustrated in the plot are great.  A few other common use cases: as a marker in leaflet: https://rstudio.github.io/leaflet/markers.html

Another compelling case I think: as labels on a plot axis (e.g. barplots of data belonging to different species) or the legend (e.g. in a lineplot of data from different species).  In both cases it might be nice to have options with or without additional text.

**Comment by sckott** [comment]
thanks for input @cboettig 

I like those use cases: 

- [ ] a leaflet marker
- [ ] labels on a plot axis or the legend

**Comment by sckott** [comment]
sorry about the very long wait on this @pssguy - i guess it's a side side side project

there's a new fxn `save_png` to save a png to disk

there's a new fxn `plot_phylopic_base` to plot a standalone png, but it's not qutie working yet

i added an example of how to use the png's as icons in leaflet maps, though nothing added to the pkg for it

didn't add anything or examples yet for the "labels on a plot axis or the legend" use case
